courses/machine_learning/intro_to_tensorflow/2_dataset_api.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>typo, missing 's': "..exact number of element.." to "..exact number of elements.."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done!</p>